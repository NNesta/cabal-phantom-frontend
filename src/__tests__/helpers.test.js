import IconButton from '../components/dashboard/helpers/IconButton';
import ContentHeader from '../components/dashboard/helpers/ContentHeader';
import React from 'react';
import { render, fireEvent, cleanup, waitFor } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/store';
import userEvent from '@testing-library/user-event';
import { BrowserRouter as Router, Link } from 'react-router-dom';
import { GitHub } from '@material-ui/icons';


const TopBarComponent = () => {
	return render(
		<Provider store={store}>
				<Router>
					<IconButton />
				</Router>
		</Provider>
	);
};


describe('TopBar component', () => {
    afterEach(cleanup)

    it('Should render add user component properly', () => {
		const { asFragment } = TopBarComponent();
		expect(asFragment(<IconButton />)).toMatchSnapshot();
    });
    
})
