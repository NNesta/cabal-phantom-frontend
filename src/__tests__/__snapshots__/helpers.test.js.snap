// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar component Should render add user component properly 1`] = `
<DocumentFragment>
  <div
    class="button_icon"
  >
    <button
      class="btn btn-secondary"
    >
      <div />
      <div
        class="ml-2"
      />
    </button>
  </div>
</DocumentFragment>
`;
